modules/1_prepare/variables.tf

```diff
@@ -23,8 +23,8 @@ variable "cluster_id" {
   type = string
 
   validation {
-    condition     = length(var.cluster_id) <= 14
-    error_message = "Length cannot exceed 14 characters when combined with cluster_id_prefix."
+    condition     = length(var.cluster_id) <= 24
+    error_message = "Length cannot exceed 24 characters when combined with cluster_id_prefix."
```

Comment:
@cs-zhang overall looks good. Can you please update the docs as well to indicate the change

Reply:
@cs-zhang I am not clear about the length aspect, can you please share the reference doc where this is mentioned (length <= 24)

Reply:
There is no doc for the name length limitation, but the name length for pi_network_name is limited to 32, its name created by `cluster_id` plus `-pub-net`, if over this limit, the TF will throw "Error: Failed to Create PI Network".

Reply:
Keep `-pub-net ` is to show these networks are created by TF automation.  It can be removed if we don't care to distinguish them from other networks.

Reply:
I don't see the value of keeping -pub-net - isn't everything created by TF automation?  There may be other networks in the service area but they are probably named for human readability and would be hard to confuse with an automatically created network?  I'd prefer as many characters as possible for these names, and 32 is much better than 24 or 14.